pkg/controller/instance/instance_controller_test.go

```diff
@@ -6,14 +6,20 @@ import (
 	"context"
 	"fmt"
 	"log"
+	"os"
+	"path/filepath"
 	"sync"
 	"testing"
 	"time"
 
+	"github.com/kudobuilder/kudo/pkg/apis"
```

Comment:
Move this to the section below.

Reply:
😞 this import grouping start to annoy me